packages/sdk/client/src/diagnostics/platform.ts

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright 2022 DXOS.org
+//
+
+export type Platform = {
+  type: 'browser' | 'node';
+  platform: string;
+  runtime?: string;
+};
+
+export const getPlatform = async (): Promise<Platform> => {
+  if (typeof window !== 'undefined') {
+    const { userAgent } = window.navigator;
+    return {
+      type: 'browser',
+      platform: userAgent,
+    };
+  }
+
+  // https://nodejs.org/api/os.html
+  try {
+    const { machine, platform, release } = await require('node:os');
```

Comment:
no need to await require
```suggestion
    const { machine, platform, release } = require('node:os');
```

Reply:
Strange lint complained. Needed to add:
eslint-disable-next-line @typescript-eslint/no-var-requires